html/components/card.stories.js

```diff
@@ -80,16 +80,17 @@ export const highlightedHeader = () =>
           class="
             sprk-b-TypeDisplaySeven
             sprk-o-Stack__item
-            sprk-u-Color--white
+            sprk-c-Card__highlighted-heading
           "
         >
           Description
         </h3>
 
         <h4
           class="
-            sprk-b-TypeDisplayFive sprk-o-Stack__item
-            sprk-u-Color--white
+            sprk-b-TypeDisplayFive
+            sprk-o-Stack__item
+            sprk-c-Card__highlighted-heading
```

Comment:
I think this class and sass var is particularly beneficial for our consumers because it allows them to change out the highlighted header card styles easily via the var which I think we might have been asked about before.